crates/grafbase-sdk/src/component/authentication.rs

```
@@ -15,4 +15,12 @@ impl wit::AuthenticationGuest for Component {
             result.map(Into::into).map_err(Into::into)
         })
     }
+
+    fn public_metadata() -> Vec<wit::PublicMetadataEndpoint> {
+        let Ok(extension) = state::extension() else {
```

better to raise an error, it should never happen